Scoop/Controller/Profile/EditProfileViewController.swift

```diff
@@ -623,3 +624,98 @@ class EditProfileViewController: UIViewController {
     }
     
 }
+
+// MARK: - UITextFieldDelegate
+
+extension EditProfileViewController: UITextFieldDelegate {
+    func textField(_ textField: UITextField, shouldChangeCharactersIn range: NSRange, replacementString string: String) -> Bool {
+        return !(textField == classTextField || textField == pronounsTextField)
+    }
+    
+    func textFieldDidBeginEditing(_ textField: UITextField) {
+        if let onboardingTextField = textField as? OnboardingTextField {
+            if let associatedView = onboardingTextField.associatedView as? LabeledTextField {
+                associatedView.labeledTextField(isSelected: true)
+                associatedView.hidesLabel(isHidden: false)
+            }
+        }
```

Comment:
Try this. Not sure if it works though.
```suggestion
        if let onboardingTextField = textField as? OnboardingTextField,
           let associatedView = onboardingTextField.associatedView as? LabeledTextField {
                associatedView.labeledTextField(isSelected: true)
                associatedView.hidesLabel(isHidden: false)
            }
        }
```